src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml

```diff
@@ -8,3 +8,6 @@ imports:
     - { resource: '@SyliusCoreBundle/Resources/config/app/fixtures/dress.yaml' }
     - { resource: '@SyliusCoreBundle/Resources/config/app/fixtures/jeans.yaml' }
     - { resource: '@SyliusCoreBundle/Resources/config/app/fixtures/custom_fixtures.yaml' }
+
+parameters:
+    env(HOSTNAME): 'localhost'
```

Comment:
This might cause unwanted conflicts if someone is already using `HOSTNAME` environment variable. Could we make it `SYLIUS_FIXTURES_HOSTNAME` instead?